File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-2.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        pos = 1

        for i in range(len(nums) - 1):
            if nums[i] == nums[i + 1]:
                continue
            elif nums[i] != nums[i + 1]:
                nums[pos] = nums[i+1]
                pos += 1

        return pos

File: Data Structures & Algorithms/concatenation-of-array/submission-2.java
class Solution {
    public int[] getConcatenation(int[] nums) {

        int n = nums.length;
        int arrSize = n * 2;
        int[] ans = new int[arrSize];

        for (int i = 0; i < n; i++){
            ans[i] = ans[i + n] = nums[i];
        }
        

        return ans;
    }
}